server/domain/routers/dashboard.go

```diff
@@ -0,0 +1,31 @@
+package routers
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"school-project-2019/server/domain/devices"
+	"github.com/julienschmidt/httprouter"
+)
+
+func DashboardInit(router *httprouter.Router) {
+	router.GET("/sensors", GetAllSensors)
+}
+
+func GetAllSensors(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	sensor := devices.Sensor{}
+	sensors, err := sensor.FindManySensors()
+	// testing custom error response
+	if err == devices.NOT_FOUND {
+		http.Error(w, errors.New("no sensors in the database").Error(), http.StatusNotFound)
+		return
+	}
+
+	response, err := json.Marshal(sensors)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
```

Comment:
Better to send internal server error status in code

Reply:
i cant remember that i wrote this file, anyway ok:)